src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -942,45 +874,13 @@ public String toVariableNameString(InventoryHolder holder) {
 						return toString(holder, 0);
 					}
 				}));
-		Classes.registerClass(new ClassInfo<>(GameMode.class, "gamemode")
+		Classes.registerClass(new EnumClassInfo<>(GameMode.class, "gamemode", "game modes")
 				.user("game ?modes?")
 				.name("Game Mode")
 				.description("The game modes survival, creative, adventure and spectator.")
-				.usage("creative/survival/adventure/spectator")
 				.examples("player's gamemode is survival",
 						"set the player argument's game mode to creative")
-				.since("1.0")
-				.defaultExpression(new SimpleLiteral<>(GameMode.SURVIVAL, true))
```

Comment:
The default expression here is different than the event value expression. Needs to be defined in the arguments for the EnumClassInfo constructor to be survival as default, thus making it not removed.

`new EnumClassInfo<>(GameMode.class, "gamemode", "game modes", new SimpleLiteral<>(GameMode.SURVIVAL, true))`

Reply:
I made sure that all classes retained their default expression, this one must've slipped mb